benches/README.md

```diff
@@ -3,79 +3,197 @@
+  1 (1.00%) low severe
   2 (2.00%) low mild
   2 (2.00%) high mild
-F128p18/next_power_of_ten
-                        time:   [5.8670 ns 5.8787 ns 5.8895 ns]
+  4 (4.00%) high severe
+F128p18/FixedPoint::try_from(f64) deviation on MAX
+                        time:   [5.5499 us 5.5503 us 5.5507 us]
+Found 13 outliers among 100 measurements (13.00%)
+  1 (1.00%) low severe
+  3 (3.00%) high mild
+  9 (9.00%) high severe
+F128p18/FixedPoint::try_from(f64) ~10^-12
+                        time:   [127.28 ns 128.17 ns 129.01 ns]
+Found 19 outliers among 100 measurements (19.00%)
+  1 (1.00%) high mild
+  18 (18.00%) high severe
+F128p18/FixedPoint::try_from(f64) ~0.1
+                        time:   [59.345 ns 59.440 ns 59.552 ns]
+Found 13 outliers among 100 measurements (13.00%)
+  5 (5.00%) high mild
+  8 (8.00%) high severe
+F128p18/FixedPoint::try_from(f64) ~1e6
+                        time:   [29.304 ns 29.395 ns 29.500 ns]
+Found 9 outliers among 100 measurements (9.00%)
+  6 (6.00%) high mild
+  3 (3.00%) high severe
 
-F64p9/cadd              time:   [457.84 ps 458.13 ps 458.46 ps]
-F64p9/rmul              time:   [455.02 ps 455.34 ps 455.70 ps]
+F64p9/cadd              time:   [459.11 ps 459.14 ps 459.17 ps]
+Found 18 outliers among 100 measurements (18.00%)
+  2 (2.00%) low severe
+  7 (7.00%) low mild
+  5 (5.00%) high mild
+  4 (4.00%) high severe
+F64p9/rmul              time:   [458.48 ps 458.52 ps 458.55 ps]
 Found 9 outliers among 100 measurements (9.00%)
-  9 (9.00%) high mild
-F64p9/rdiv              time:   [437.68 ps 438.00 ps 438.36 ps]
-Found 1 outliers among 100 measurements (1.00%)
-  1 (1.00%) high mild
+  2 (2.00%) low mild
+  3 (3.00%) high mild
+  4 (4.00%) high severe
+F64p9/rdiv              time:   [445.60 ps 445.65 ps 445.70 ps]
+Found 8 outliers among 100 measurements (8.00%)
+  1 (1.00%) low severe
+  7 (7.00%) high severe
 F64p9/rsqrt (~10^4, precise cases)
-                        time:   [38.846 ns 38.885 ns 38.926 ns]
-Found 2 outliers among 100 measurements (2.00%)
-  2 (2.00%) high mild
-F64p9/rsqrt (deviation) time:   [93.532 ns 93.651 ns 93.764 ns]
+                        time:   [40.895 ns 40.916 ns 40.937 ns]
 Found 3 outliers among 100 measurements (3.00%)
-  2 (2.00%) low mild
+  1 (1.00%) low mild
+  2 (2.00%) high severe
+F64p9/rsqrt (deviation) time:   [90.416 ns 90.510 ns 90.606 ns]
+Found 4 outliers among 100 measurements (4.00%)
+  4 (4.00%) high severe
+F64p9/rsqrt (adaptive)  time:   [87.880 ns 87.927 ns 87.975 ns]
+Found 22 outliers among 100 measurements (22.00%)
+  1 (1.00%) low mild
   1 (1.00%) high mild
-F64p9/rsqrt (adaptive)  time:   [89.743 ns 89.795 ns 89.839 ns]
-Found 12 outliers among 100 measurements (12.00%)
-  2 (2.00%) low severe
-  6 (6.00%) low mild
+  20 (20.00%) high severe
+F64p9/next_power_of_ten time:   [2.4977 ns 2.5153 ns 2.5343 ns]
+F64p9/FixedPoint::try_from(f64) deviation on MIN_POSITIVE
+                        time:   [1.7986 us 1.7988 us 1.7990 us]
```

Comment:
I think we need to alter implementation (to the current one?) in order to preserve performance at the same level for small numbers.